Description: Avoid dependent base class scope lookups
 These no longer work in G++ 4.7: http://gcc.gnu.org/gcc-4.7/porting_to.html
Author: Colin Watson <cjwatson@debian.org>
Bug-Debian: http://bugs.debian.org/667133
Forwarded: no
Last-Update: 2012-04-04

Index: b/cccc/cccc_tbl.cc
===================================================================
--- a/cccc/cccc_tbl.cc
+++ b/cccc/cccc_tbl.cc
@@ -96,7 +96,7 @@
   typename map_t::iterator value_iterator=map_t::find(old_item_ptr->key());
   if(value_iterator!=map_t::end())
     {
-      erase(value_iterator);
+      this->erase(value_iterator);
       retval=true;
     }
   return retval;
